### ISSUE TITLE
Jump to today on date picker

### ISSUE THREAD

**Issue by vendelin8** [flutter/flutter#72998]
## Use case
I'm running `showDatePicker`.

## Proposal
It would be awesome to have a "Jump to Today" or similar button. So if I go to other month pages, I could get back in a single click.

**Comment by justinmc** [comment]
CC @darrenaustin

**Comment by viralkachhadiya** [comment]
hello, i am working on it.

**Comment by justinmc** [comment]
Thanks!  I've assigned you.

**Comment by viralkachhadiya** [comment]
Please check the code and review my code.
I pushed the code.

**Comment by viralkachhadiya** [comment]
https://user-images.githubusercontent.com/58855915/103287770-5bc91d00-4a09-11eb-9c1f-cb4ee487ae9c.mp4

**Comment by justinmc** [comment]
Where is your code at?  Could you link to the PR?

**Comment by viralkachhadiya** [comment]
Wait, I will send you

**Comment by viralkachhadiya** [comment]
i think below link 

https://github.com/flutter/flutter/pull/73101

**Comment by viralkachhadiya** [comment]
please review my code on above pr list

**Comment by viralkachhadiya** [comment]
@justinmc
@darrenaustin
@PixelToast 
@tadaspetra
 @vendelin8

Please check below my PR link and give review for code. 

https://github.com/flutter/flutter/pull/73101

**Comment by viralkachhadiya** [comment]
I cange some code clean up and clear autoformating tool .

@justinmc 
@darrenaustin 
@PixelToast 
@tadaspetra 
@vendelin8 

Please review on below PR link

https://github.com/flutter/flutter/commit/f70a7f72eb01ccf1f8c79cb55c3db91e38a3f2d1

**Comment by vendelin8** [comment]
That's only a commit. Your PR already has those formatting changes https://github.com/flutter/flutter/pull/73101

**Comment by viralkachhadiya** [comment]
> That's only a commit. Your PR already has those formatting changes #73101

So, how to change pr with this commit

**Comment by vendelin8** [comment]
If this does the job, then you should revert the earlier commits. Or commit code to rechange the original state. When you open the PR, on the `files` tab you can check the overall change you want to merge in the PR.

**Comment by viralkachhadiya** [comment]
Hello ,

I reset the pr request .please check it.
@justinmc
@darrenaustin
@PixelToast
@tadaspetra
@vendelin8

https://github.com/flutter/flutter/pull/74377

**Comment by vendelin8** [comment]
Looks good to me. I didn't approve it, because I'm not a flutter contributor, let's wait for someone closer to the project.
Thanks Viral.

**Comment by viralkachhadiya** [comment]
@darrenaustin

please review my new above PR request

**Comment by viralkachhadiya** [comment]
@PixelToast
@TahaTesser

Can you help about my this pull request??
Please provide some guidline so I will completed it. 

https://github.com/flutter/flutter/pull/74377

**Comment by viralkachhadiya** [comment]
@vendelin8

Can you reassign me?
Because I unassigned it by my mistake

**Comment by vendelin8** [comment]
@viralkachhadiya I looked all the buttons, but I think I don't have permission to assign you.

**Comment by viralkachhadiya** [comment]
> @vendelin8
> 
> Can you reassign me?
> Because I unassigned it by my mistake

@justinmc

**Comment by justinmc** [comment]
Sorry I missed these notifications.  @viralkachhadiya do you still plan to work on this?  If so I can reassign.

**Comment by shubham3246** [comment]
Can i work on this?

**Comment by justinmc** [comment]
@shubham3246 That would be great, thanks!  I've assigned you.  Take a look at the previous attempt at a PR (https://github.com/flutter/flutter/pull/74377) if you haven't already.

**Comment by theadityaagnihotri** [comment]
Hey , I want to work on this, can you please assign it to me?

**Comment by justinmc** [comment]
@shubham3246 Looks like you have first dibs, are you still planning to open a PR for this or should I pass it to @theadityaagnihotri?

**Comment by shubham3246** [comment]
@justinmc Yeah i think its gonna take a while, so you can pass it to the other person. I'll contact again when i complete the work.
